src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,7 +55,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                 {
                     // what do we do if there's no master?
                     TargetBranch = x.FirstOrDefault(b => b.Name == "master");
-                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head.FriendlyName);
+
+                    // 'repo.Head' will be null in unit tests.
+                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head?.FriendlyName);
```

Comment:
Ah yeah - I see that I do get a mocked `IRepository` there. My bad - I hadn't investigated enough and assumed that this part of the code wasn't written testably.